modules/i18n/src/translations/si-LK/portals/console.ts

```diff
@@ -3754,6 +3768,34 @@ export const console: ConsoleNS = {
                             view: "දේශීය හිමිකම් බලන්න"
                         }
                     },
+                    sections: {
+                        manageAttributeMappings: {
+                            custom: {
+                                description: "අභිරුචි සිතියම්කරණය හරහා පරිශීලකයා පිළිබඳ තොරතුරු සන්නිවේදනය කරන්න.",
+                                heading: "අභිරුචි ගුණාංග සිතියම්ගත කිරීම"
+                            },
+                            description: "ඒපීඅයි හෝ ඔබේ යෙදුම් සමඟ අන්තර්ක්‍රියා කරන විට අස්ගාර්ඩියෝ හි ගුණාංග සිතියම්" +
+                                "ගත කර පරිවර්තනය කරන්නේ කෙසේදැයි බලන්න සහ කළමනාකරණය කරන්න.",
+                            heading: "ගුණාංග සිතියම් කළමනාකරණය කරන්න",
+                            oidc: {
+                                description: "සත්‍යාපනය සඳහා OpenID Connect භාවිතා කරන යෙදුම් සඳහා පරිශීලකයා " +
+                                    "පිළිබඳ තොරතුරු සන්නිවේදනය කරන්න.",
+                                heading: "OpenID Connect"
+                            },
+                            scim: {
+                                description: "SCIM2 ප්‍රමිතීන්ට අනුකූලව API හරහා පරිශීලකයා පිළිබඳ තොරතුරු සන්නිවේදනය කරන්න.",
+                                heading: "System for Cross-Domain Identity Management"
+                            }
+                        },
+                        manageAttributes: {
+                            attributes: {
+                                description: "සෑම ගුණාංගයකම අස්ගාර්ඩියෝ තුළ ගබඩා කර ඇති පරිශීලක දත්ත කොටසක් අඩංගු වේ.",
+                                heading: "ගුණාංග"
+                            },
+                            description: "අස්ගාර්ඩියෝට ආවේණික වූ ගුණාංග බලන්න සහ කළමනාකරණය කරන්න.",
```

Comment:
```suggestion
                            description: "WSO2 Identity Server ට ආවේණික වූ ගුණාංග බලන්න සහ කළමනාකරණය කරන්න.",
```

Merging this temporally, need to replace "WSO2 Identity Server" value dynamically